doc/sphinx/using/libraries/writing.rst

```diff
@@ -25,7 +25,8 @@ deprecated compatibility alias using :cmd:`Notation (abbreviation)`
 
    This attribute is supported by the following commands: :cmd:`Ltac`,
    :cmd:`Tactic Notation`, :cmd:`Notation`, :cmd:`Infix`, :cmd:`Ltac2`,
-   :cmd:`Ltac2 Notation`, :cmd:`Ltac2 external`.
+   :cmd:`Ltac2 Notation`, :cmd:`Ltac2 external`. To attach it to a
+   compiled library file, use :cmd:`Library Attributes`.
```

Comment:
This does not belong to the current PR but while at it, why not fix that list to add the definitions supported since https://github.com/coq/coq/pull/16890 (apparently this paragraph of the doc was missed there).

Reply:
OK, added (and soon pushed).